pkg/report/asff_test.go

```diff
@@ -0,0 +1,39 @@
+package report
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/Checkmarx/kics/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrintASFFReport(t *testing.T) {
+	tests := []struct {
+		name     string
+		caseTest jsonCaseTest
+	}{
+		{
+			name: "asff report",
+			caseTest: jsonCaseTest{
+				summary:  test.SummaryMock,
+				path:     filepath.Join(os.TempDir(), "testdir"),
+				filename: "output",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			if err := os.MkdirAll(test.caseTest.path, os.ModePerm); err != nil {
+				t.Fatal(err)
+			}
+
+			err := PrintASFFReport(test.caseTest.path, test.caseTest.filename, test.caseTest.summary)
+			require.NoError(t, err)
+
+			require.FileExists(t, filepath.Join(test.caseTest.path, "asff-"+test.caseTest.filename+".json"))
+		})
```

Comment:
Maybe we should remove the file

```suggestion
			require.FileExists(t, filepath.Join(test.caseTest.path, "asff-"+test.caseTest.filename+".json"))
			os.RemoveAll(test.caseTest.path)
		})
```

Reply:
I'm using `/tmp` dir to save files, so I think there is no need to delete them, since they are deleted automatically

Reply:
Are you sure? At least, in Windows, my files in `/tmp` seem to not be deleted automatically